trackintel/preprocessing/staypoints.py

```diff
@@ -270,12 +270,7 @@ def merge_staypoints(staypoints, triplegs, max_time_gap="10min", agg={}):
 
     # a joined dataframe sp_tpls is constructed to add the columns 'type' and 'next_type' to the 'sp_merge' table
     # concat and sort by time
-    with warnings.catch_warnings():
-        warnings.filterwarnings("ignore", message="CRS not set for some of the concatenation inputs.*")
-
-        # TODO: the warning relates to how we process when no triplegs is provided. In future versions,
-        # we want to seperate the two scenarios. See issue #463.
```

Comment:
The issue #463 concerns the seperation of the tripleg scenarios, which is not fix by this PR

Reply:
I do not understand the problem here. Empty GeoDataFrames and `Nones` are both possible to concat without a warning.

Reply:
The issue is not only about the warning - we want to change the default input of the function such that it does not require `triplegs` as input. But I think #463's title is confusing and we shall open a new issue for that

Reply:
Ah, that makes sense. Thanks for the clarification. I have added a TODO again.